src/presenced.cpp

```diff
@@ -1272,6 +1307,13 @@ void Client::removePeer(karere::Id peer, bool force)
 
     mCurrentPeers.erase(it);
 
+    // Remove peer from mPeersLastGreen map if exists
+    auto itAux = mPeersLastGreen.find(peer.val);
```

Comment:
You can directly do `mPeersLastGreen.erase(peer.val);`, it is equivalent and shorter :)